src/database/db_build.sql

```diff
@@ -9,38 +9,39 @@ CREATE TABLE countries(
 );
 CREATE TABLE teams(
   id SERIAL PRIMARY KEY ,
-  id_countries INT REFERENCES countries(id_countries),
-  team_name TEXT,
+  id_countries int,
+ FOREIGN KEY (id_countries) REFERENCES countries(id_countries),
```

Comment:
Just a small style error, needs to line up with the other fields. Can you add eslint to the project? And always eslint before you push up.

Reply:
You can add eslint in another PR.

Reply:
@jema28  added eslint